linode_api4/objects/base.py

```diff
@@ -277,6 +277,8 @@ def save(self, force=True) -> bool:
                 ):
                     data[key] = None
 
+            # Ensure we serialize any values that may not be already serialized
+            data = _flatten_request_body_recursive(data)
```

Comment:
User-defined unpopulated values were not being flattened here previously, causing an error in cases like this:

```python
pool = LKENodePool(client, 456, 123)
pool.taints = [LKENodePoolTaint(...)]
pool.save()
```